crates/linker/src/pe.rs

```diff
@@ -1276,6 +1285,40 @@ pub(crate) fn redirect_libc_functions(name: &str) -> Option<&str> {
     }
 }
 
+// 0000000000000000 <.text>:
+//    0:	51                   	push   rcx
+//    1:	50                   	push   rax
+//    2:	48 3d 00 10 00 00    	cmp    rax,0x1000
+//    8:	48 8d 4c 24 18       	lea    rcx,[rsp+0x18]
+//    d:	72 18                	jb     27 <.text+0x27>
+//    f:	48 81 e9 00 10 00 00 	sub    rcx,0x1000
+//   16:	48 85 09             	test   QWORD PTR [rcx],rcx
+//   19:	48 2d 00 10 00 00    	sub    rax,0x1000
+//   1f:	48 3d 00 10 00 00    	cmp    rax,0x1000
+//   25:	77 e8                	ja     f <.text+0xf>
+//   27:	48 29 c1             	sub    rcx,rax
+//   2a:	48 85 09             	test   QWORD PTR [rcx],rcx
+//   2d:	58                   	pop    rax
+//   2e:	59                   	pop    rcx
+//   2f:	c3                   	ret
```

Comment:
@brian-carroll was able to track down the source https://codebrowser.dev/llvm/compiler-rt/lib/builtins/x86_64/chkstk.S.html, compiled that with zig and just pasted in the bytes here. But yes this is now x86_64 specific. 

To my knowledge windows on ARM is not very mature yet, so we'll have to see how that develops.

Reply:
Yeah that's also my understanding.
But if this is the only CPU-specific thing, then it's pretty easy to insert some conditional for it when we need to. And if we ever need more chunks of bytes like this, we could create a struct for them.